how-we-work/oncall-responsibilities.md

```diff
@@ -20,13 +20,13 @@ An on-call rotation begins on Monday at 9am PT. Each week, there is a primary an
 
 ##### Deployment summary table
 
-|                   | dev                            | staging                                                                                  | prod                                                                                     | dev-canary                                                |
-| ----------------- | ------------------------------ | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------------------------------------------- |
-| single-cell-infra | TFE auto                       | TFE auto                                                                                 | TFE manual                                                                               | TFE manual                                                |
-| data-portal       | Need to set up auto deployment | Github Action Auto Deploy                                                                | happy/script Deploy Manual                                                               | no deployment                                             |
-| explorer          | Github Action Auto Deploy      | single-cell-infra deploy script to kick off github action (also possible to run locally) | single-cell-infra deploy script to kick off github action (also possible to run locally) | Github Action Auto Deploy (does not auto rebase off main) |
+| repo                    | dev                            | staging                                                                                  | prod                                                                                     | dev-canary                                                |
```

Comment:
Can we remove the `dev-canary` column since the CC team no longer needs their own deployment?

Reply:
done!